src/swagger.json

```diff
@@ -9,19 +9,59 @@
   "produces": [
     "application/json"
   ],
-  "securityDefinitions": {
-    "app-jwt": {
-      "type": "apiKey",
-      "in": "header",
-      "name": "Authorization"
-    }
-  },
-  "security": [
-    {
-      "app-jwt": []
-    }
-  ],
   "paths": {
+    "/auth": {
+      "post": {
+        "tags": [
+          "Auth"
+        ],
+        "summary": "User Login",
+        "description": "",
+        "operationId": "userLogin",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "payload",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "email": {
+                  "type": "string"
+                },
+                "password": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
I think it's more proper to use status code 302 instead of 200 since it is redirect

Reply:
Fixed.